### PR TITLE
[Refactor API] r(b): carto.source

### PR DESCRIPTION
**Pull request by manmorjim**
- Creating the namespace `carto.source`
- Dropping 'Source suffix' just in public API:
  - `carto.source.DO`
  - `carto.source.GeoJSON`
  - `carto.source.SQL`
  - `carto.source.Dataset`

### CODE REVIEW

File: src/lib/index.ts [hunk 1]
```diff
@@ -43,8 +43,15 @@ export const auth = {
   setDefaultCredentials
 };
 
+// carto.source
+export const source = {
+  DO: DOSource,
```
Comment (VictorVelarde) [inline src/lib/index.ts]: We won't expose this DO yet, you can keep it commented and with a note

File: src/lib/source/GeoJSONSource.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { GeoJSON, GeoJsonGeometryTypes, Feature } from 'geojson';
+import { GeoJSON, Feature, GeoJsonGeometryTypes } from 'geojson';
```
Comment (VictorVelarde) [inline src/lib/source/GeoJSONSource.ts]: It's curious how they use GeoJSON and GeoJson in the same lib... 😄

**Review by VictorVelarde — COMMENTED** [review]
Overall, it looks really good to me. I feel like the result is more clear and the 3 commits approach made the review easier, to thx for it. Let's wait a bit for geojson and histogram dataviews to be merged, and then I'll approve this

File: src/lib/source/GeoJSONSource.ts (hunk 2 above)
Comment (manmorjim) [reply src/lib/source/GeoJSONSource.ts]: yup, I thought the same... _why did they do that?_ :shrug:

**Review by VictorVelarde — APPROVED** [review]
👌 . Nice, let's go ahead!

**Pull request merged**